sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_base_client.py

```diff
@@ -79,3 +116,32 @@ def __init__(self, endpoint: str, credential: Union[AzureKeyCredential, TokenCre
         )
         self._deserialize = _get_deserialize(self._api_version)
         self._generated_models = self._client.models(self._api_version)
+
+    @distributed_trace
+    def send_request(self, request: HttpRequest, *, stream: bool = False, **kwargs) -> HttpResponse:
+        """Runs a network request using the client's existing pipeline.
+
+        The request URL can be relative to the base URL. The service API version used for the request is the same as
+        the client's unless otherwise specified. We only support service API version 2022-08-31 and later in request.
```

Comment:
> We only support service API version 2022-08-31 and later in request.

I would consider rewording this. To me, it implies that `send_request` only supports API versions 2022-08-31 and later -- maybe something like "Overriding the client's configured API version is only supported on API version 2022-08-31 and later" would be more accurate.